ES/ESmy021.xml

```diff
@@ -40,62 +40,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      </note>
 
                         <msItem xml:id="ms_i1.1">
-                        <locus from="1va" to="3vb"/>
+                        <locus from="1va" to="3va"/>
                         <title type="complete" corresp="LIT4860Litanies"/>
-                        <incipit>
-                        <q xml:lang="gez"><hi rend="rubric">ሊጦን፡ ዘነግህ፡ ወካዕበ፡ ናስተበቍዕ፡ ዘኵ</hi>ሎ፡ ይእኅዝ፤ እግዚአብሔር፤ አብ፤ ለእግዚእ፤
+                        <incipit xml:lang="gez">
+                        <locus target="#1va"/><hi rend="rubric">ሊጦን፡ ዘነግህ፡ ወካዕበ፡ ናስተበቍዕ፡ ዘኵ</hi>ሎ፡ ይእኅዝ፤ እግዚአብሔር፤ አብ፤ ለእግዚእ፤
                           ወመድኃኒነ፤ ኢየሱስ፤ ክርስቶስ፤ እንዘ፤ ነአኵቶ፤ እስመ፡ ዓቀበነ፡ በኑኃ፡ ሌሊተ፤ ወአብጽሐነ፤ እምጽልመት፤ ውስተ፤ ወብርሃን፤
-                          እሙስና፤ ውስተ፤ ዘኢይማስን፤ እምኢያዓእምሮ፤ ውስተ፡ አእምሮ፡ አማን፤ ከመ፡ በኑኃ፤ ሌሊትኒ፡ በኵሎ፡ ሰላም፨</q>
+                          እሙስና፤ ውስተ፤ ዘኢይማስን፤ እምኢያዓእምሮ፤ ውስተ፡ አእምሮ፡ አማን፤ ከመ፡ በኑኃ፤ ሌሊትኒ፡ በኵሎ፡ ሰላም፨
                         </incipit>
                       </msItem>
 
                       <msItem xml:id="ms_i1.2">
                       <locus from="3va" to="3vb"/>
-                      <title type="complete" corresp="LIT4711Mastabqwe#ZaMutan"/>
-                      <incipit>
-                      <q xml:lang="gez"><hi rend="rubric">ወካዕበ፡ በእንተ፡ እለ፤</hi> ኖሙ፤ አኀዊነ፤
+                      <title type="complete" corresp="LIT4711Mastabqwe#SupplicationDead"/>
```

Comment:
all title elements here and elsewhere in the MS need `@ref` and not `@corresp`

Reply:
sorry, stupid mistake